Sources/Kitura/RouterResponse.swift

```diff
@@ -121,6 +121,20 @@ public class RouterResponse {
         statusCode = .unknown
     }
 
+    deinit {
+        if !state.invokedEnd {
+            if !state.invokedSend && statusCode == .unknown {
```

Comment:
I don't see a need to test again for !state.invokeSend. We are the deinit. No one is suppose to have a reference.